controllers/__snapshots__/aliases.test.js.snap

```diff
@@ -6402,5 +6402,85 @@ Array [
     "from": "/welsh/wales/about/customer-service/supplier-zone/contracts-finder",
     "to": "/welsh/about/customer-service/supplier-zone",
   },
+  Object {
+    "from": "/-/media/Images/Logos/JPEGs/hi_big_e_min_blue.jpg",
+    "to": "/assets/images/logos/tnlcf/monolingual/colour.png",
+  },
+  Object {
+    "from": "/welsh/-/media/Images/Logos/JPEGs/hi_big_e_min_blue.jpg",
+    "to": "/welsh/assets/images/logos/tnlcf/monolingual/colour.png",
+  },
+  Object {
+    "from": "/england/-/media/Images/Logos/JPEGs/hi_big_e_min_blue.jpg",
+    "to": "/assets/images/logos/tnlcf/monolingual/colour.png",
+  },
+  Object {
+    "from": "/welsh/england/-/media/Images/Logos/JPEGs/hi_big_e_min_blue.jpg",
```

Comment:
We probably don't need all the regional redirects here. One for a bit of future clean up I reckon to make the region stuff opt-in. Fine for now though.